examples/daemon_with_loader/src/main.rs

```diff
@@ -26,12 +25,6 @@ Open a new terminal without exiting this program, and run:
 - `help` to list all available commands. 
 "#
     );
-    use std::io::{Read, Write};
-    let mut stdin = std::io::stdin();
-    let mut stdout = std::io::stdout();
 
-    write!(stdout, "Press any key to exit...").unwrap();
-    stdout.flush().unwrap();
-
-    let _ = stdin.read(&mut [0u8]).unwrap();
```

Comment:
The print above this says to use the CLI to inspect metadata, but if there is no pause in execution then the daemon won't be running when statring the CLI.

Reply:
this code was never run before (including that println), the `game::run()` call blocks in a loop.  also I verified that the functionality in that println worked (both with deleting the .assets_db and meta files, and without).

Reply:
Oh, great

Reply:
I found out what caused the examples to not finish properly. I had inadvertently changed the .meta format, which caused deserialization to fail. I wanted to address this more robustly, so I removed cached, derived data from the .meta file entirely, as well as improved the handling of versioning of the format. These changes are now on master

Reply:
Could you revert this please? I will merge once it's done

Reply:
examples reverted (other than clippy fixes).  I both of them.  Meta files were updated so i committed those too.